cypress/e2e/course/createEditStopSyncedCourse.feature

```diff
@@ -56,6 +58,51 @@ Feature: Teacher can create, update and stop a synchronized course
       | teacher1_nbc | nbc   | Cypress-Test-Group1             | Karl Herzog  | Herbert Kraft | 01.08.2024 | 31.07.2025 |
       | teacher1_nbc | nbc   | Cypress-Test-Group1             | Karl Herzog  | Herbert Kraft | 01.08.2024 | 31.07.2025 |
 
+  Scenario Outline: Create a non-synchronized course
+    Given I am logged in as a '<user>' at '<state>'
+    When I go to rooms overview
+    When I click on FAB to create a new room depending on sub menu
+    Then I see section one area on the course create page
```

Comment:
The course create dialog consists of several pages / sections / areas. This is the first of those called here "section one area"...

This is used in 16 .feature files... It would not be a good practice to have a new key doing the same as an existing one, but one could refactor all 16 files including current stable tests, which means allignments with qa ;) 

For now i will refrain from doing that, but i've noted it and will forward it to the next qa weekly.